cmd/admin-handlers.go

```diff
@@ -240,18 +240,20 @@ func (a adminAPIHandlers) ServerInfoHandler(w http.ResponseWriter, r *http.Reque
 	if objectAPI == nil {
 		return
 	}
-
-	thisAddr, err := xnet.ParseHost(GetLocalPeer(globalEndpoints))
-	if err != nil {
-		writeErrorResponseJSON(ctx, w, toAdminAPIErr(ctx, err), r.URL)
-		return
+	hostName := r.Host
```

Comment:
could we move this to a function since it is being used multiple times?

Reply:
should this be done in another iteration? @krisis

Reply:
ok,done